virttest/virsh.py

```diff
@@ -642,15 +642,15 @@ def __str__(self):
 class EventTracker(object):
 
     @staticmethod
-    def start_get_event(vm_name):
+    def start_get_event(vm_name, event_cmd='event %s --all --loop'):
```

Comment:
event_cmd --> event_options, it looks like 'virsh event' is mandatory for any event action, and event_options e.g --all-loop is flexible, and will affect virsh event behaviour.

Reply:
Hi chunfu, Thanks  for your review. But we have met this event cmd ` "qemu-monitor-event vm_name --loop" `in some cases, so  'virsh event' may not be mandatory.